### ISSUE TITLE
Heredocs with other method params on the same line break highlighting

### ISSUE THREAD

**Issue by segiddins** [rubyide/vscode-ruby#183]
### Your environment

- `vscode-ruby` version: 0.12.1
- Ruby version: 2.4.1
- VS Code version: 1.14.1
- Operating System: macOS 10.12.5
- Hardware (optional): 

Make sure you have `ruby`, `ruby-debug-ide` and `ruby-debug-basex19` installed before submitting your issue -- thank you !

### Expected behavior

The parameters after the heredoc-begin, plus the method called on the heredoc string, should be highlighted normally

### Actual behavior

They're all highlighted as a string

### Steps to reproduce the problem

paste the following into a ruby file:

```ruby
#!/usr/bin/env ruby

function_call <<-EOS.strip, false, nil, :foo => :bar
  the quick
EOS
```

**Comment by papa-whisky** [comment]
This might be fixed in VSCode 1.17 ? ([this bit](https://code.visualstudio.com/updates/v1_17#_improved-colorization-in-dark-and-light-themes), I haven't actually tested it sorry)

**Comment by wingrunr21** [comment]
Closing for issue cleanup. Apologies if this is still an issue. We are working to improve the core extension experience.
